components/crud-web-apps/README.md

```diff
@@ -0,0 +1,8 @@
+# Build instructions for docker images
+To build the Jupyter and Tensorboards docker images directly (without the Makefile) for development purposes, the following commands can be used from this directory.
```

Comment:
I don't like the phrasing of this sentence, as it implies that for development purposes we need to run the command directly and can't use the Makefifle.

We can use the Makefile to build Jupyter app's image by running
```bash
REGISTRY_PROJECT=my-repo make docker-build
```

[ Tensorboard's makefile uses slightly different vars but we should iron them out and have the same template when we setup the CI/CD ]